docs/use-the-network/console/integrations/google-sheets.mdx

```diff
@@ -0,0 +1,114 @@
+---
+id: google-sheets
+hide_title: true
+sidebar_label: Google Sheets
+slug: /use-the-network/console/integrations/google-sheets
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+# Google Sheets
+
+## Creating a Form
+
+[Google Forms](https://forms.google.com)
+
+Create a form.
+Put in your fields, we're going to an environment sensor as an example.
```

Comment:
This sentence is a bit confusing. I think we're missing some words?

Reply:
Good catch, thank you. I've updated some other verbiage as well.